src/new/templates/rust/ui/hyperapp-skeleton/README.md

```diff
@@ -0,0 +1,251 @@
+# Hyperware Skeleton App
+
+A minimal, well-commented skeleton application for the Hyperware platform using the Hyperapp framework.
+This skeleton provides a starting point for building Hyperware applications with a React/TypeScript frontend and Rust backend.
+
+Either prompt your favorite LLM directly with instructions on how to build your app or add them to `instructions.md`!
+
+Recommended usage:
+- Clone this repo & clean up git-related stuff:
+  ```bash
+  git clone https://github.com/humanizersequel/hyperapp-skeleton.git
+  cd hyperapp-skeleton
+  rm -rf .git
+  ```
```

Comment:
update this step above to something more kit-like. perhaps something like

"""
- Create this template
  ```bash
  kit new foo --template hyperapp-skeleton --ui
  cd foo
  ```
"""